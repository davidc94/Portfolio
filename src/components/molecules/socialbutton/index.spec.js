import React from "react"
import renderer from "react-test-renderer"
import SocialButton from "./index";

describe("SocialButton", () => {
  it("renders as expected with github", () => {
    expect(renderer.create(<SocialButton social='github'/>).toJSON()).toMatchSnapshot();
  })
  it("renders as expected with linkedin", () => {
    expect(renderer.create(<SocialButton social='linkedin'/>).toJSON()).toMatchSnapshot();
  })
})