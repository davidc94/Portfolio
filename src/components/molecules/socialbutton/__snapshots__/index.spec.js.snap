// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialButton renders as expected with github 1`] = `
<a
  href="https://github.com/davidc94"
  target="_blank"
>
  <div
    className="sc-bdVaJa ekhORe"
  >
    <img
      className="sc-bwzfXH lkoPVR"
      src=""
    />
  </div>
</a>
`;

exports[`SocialButton renders as expected with linkedin 1`] = `
<a
  href="https://www.linkedin.com/in/davidchellapah/"
  target="_blank"
>
  <div
    className="sc-bdVaJa ekhORe"
  >
    <img
      className="sc-bwzfXH lkoPVR"
      src=""
    />
  </div>
</a>
`;
